src/main/scala/org/opennetworkinsight/LDA.scala

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
No special reason, I think that's how it was working before with Dispatcher. 
Are you saying we need to use -DSOURCE? and not use args(0)?